aspnetcore/blazor/get-started.md

```diff
@@ -28,19 +28,20 @@ Get started with Blazor:
 
    1\. Install the latest [Visual Studio preview](https://visualstudio.com/vs/preview) with the **ASP.NET and web development** workload.
 
-   2\. Install the latest [Blazor extension](https://go.microsoft.com/fwlink/?linkid=870389) from the Visual Studio Marketplace. This step makes Blazor templates available to Visual Studio.
+   2\. Create a new project.
 
-   3\. Create a new project.
+   3\. Select **ASP.NET Core Web Application**. Select **Next**.
```

Comment:
Blazor now shows up top level in the new project dialog. It shows up as "Blazor Server App" in 16.3 P1. It will change to just "Blazor App" in 16.3 P2. Let's point folks to that instead.